### PR TITLE
[DOCKER] Add docker demo image

### PR DESCRIPTION
**Pull request by tqchen**
cc @tmoreau89

### CODE REVIEW

File: docs/install/docker.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. _docker-images:
+
+Docker Images
+=============
+We provide several prebuilt docker images to quickly try out tvm.
+These images are also helpful to do demo/tutorials of TVM.
```
Comment (tmoreau89) [inline docs/install/docker.rst]: "do demo/tutorials of TVM" -> "run through TVM demo and tutorials"

File: docs/install/from_source.rst [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+- We can then build tvm and related libraries.
+
+  .. code:: bash
+
+      cd build
+      cmake ..
+      make -j4
+
+If everything goes well, we can go to the specific language installation section.
```
Comment (tmoreau89) [inline docs/install/from_source.rst]: Add reference/hyperlink

Comment (tqchen) [reply docs/install/from_source.rst]: sorry I merged this in before seeing your comment, can you send a PR to add the fix you mentioned?

Comment (tmoreau89) [reply docs/install/from_source.rst]: these are minor cosmetic changes to the doc - so I won't be PRing those.

**Pull request merged**